packages/backend/src/app.ts

```diff
@@ -448,11 +454,12 @@ export class App {
 
     koa.use(
       koaMiddleware(this.apolloServer, {
-        context: async (): Promise<TenantedApolloContext> => {
+        context: async (): Promise<ForTenantIdContext> => {
           return {
             ...tenantApiSignatureResult,
             container: this.container,
-            logger: await this.container.use('logger')
+            logger: await this.container.use('logger'),
+            forTenantId: this.config.operatorTenantId
```

Comment:
Including the operator id by default feels less safe, we should either let it be undefined (and just declare this as the `TenantedApolloContext`, or have `ForTenantIdContext` let `tenantId` be undefined)